src/exec/handle_redirect.c

```diff
@@ -39,13 +39,13 @@ static int	open_redirect_file(t_redirect_info *info, t_ctx *ctx)
 	return (fd);
 }
 
-static const char	*expand_filepath(t_list *redirects, t_ctx *ctx)
+static const char	*expand_filepath(t_list *redirect, t_ctx *ctx)
```

Comment:
```suggestion
static const char	*expand_filepath(const char *filepath, t_ctx *ctx)
```

Reply:
I've updated the function. Please review it!